src/yunohost/data_migrations/0009_dh_params.py

```diff
@@ -0,0 +1,28 @@
+import re
+import os
+
+from moulinette.utils.log import getActionLogger
+from moulinette.utils.filesystem import chown
+
+from yunohost.tools import Migration
+from yunohost.service import service_regen_conf
+
+command = "nice -n 19 openssl dhparam -out /etc/ssl/private/dh2048.pem -outform PEM -2 2048 -dsaparam 2> /var/log/yunohost/dhparam_generation.log && chown root:ssl-cert /etc/ssl/private/dh2048.pem && yunohost service regen-conf >> /var/log/yunohost/dhparam_generation.log && rm /etc/cron.hourly/yunohost-generate-dh-params\n"
+dhparams_file = "/etc/ssl/private/dh2048.pem"
+
+class MyMigration(Migration):
+    "This migration will add dh_params line and generate it in installed instance"
+
+    def migrate(self):
+
+        if os.path.exists(dhparams_file) is False:
```

Comment:
```suggestion
        if not os.path.exists(dhparams_file):
```

Proper way of writing this kind of condition in python.